static/src/javascripts/projects/common/modules/spacefinder.js

```diff
@@ -60,8 +58,7 @@ const LOADING_TIMEOUT = 5000;
 const defaultOptions: SpacefinderOptions = {
     waitForImages: true,
     waitForLinks: true,
-    waitForInteractives: false,
-    waitForAds: false,
```

Comment:
Have you removed the calls to spacefinder/space-filler calls that use this flag?